cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -30,6 +30,8 @@
 from xmodule.modulestore.exceptions import ItemNotFoundError
 from xmodule.modulestore.django import modulestore
 from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory, LibraryFactory
+from student import auth
```

Comment:
Nit: probably better to just import what you need: `from student.auth import add_users`.